src/translate.js

```diff
@@ -84,20 +84,35 @@ function uploadRemoteTranslations(auth, languages, translations) {
     rows.push([k, ...languages.map((l) => translation[l])]);
   });
 
-  log('Uploading new translations');
+  log("Uploading new translations");
   return uploadSheetData(auth, rows);
 }
 
 function exportTranslations(translations) {
-  fs.writeFileSync(
-    config.translationsPath,
-    JSON.stringify(translations, null, 2),
-  );
+  if (config.locales) {
+    config.locales.forEach((locale) => {
+      fs.writeFileSync(
+        `${locale}.json`,
+        JSON.stringify(
+          config.flatten
+            ? unflatten(translations[locale])
+            : translations[locale],
+          null,
+          2
+        )
+      );
+    });
+  } else {
+    fs.writeFileSync(
```

Comment:
With this condition, I assume its generating a different locale template ? How about we configure `locales` to be a required field in config and remove this condition entirely.

After this `gtranslate` would split file according to `locales` by default. 
(If dev would want to have it separate way we can accept some callback function in config allowing them full control over that. We can support that callback function later though)

Reply:
Ok, I got your point here